__tests__/eslintPlugin/noAnimatedStyleToNonAnimatedComponent.test.tsx

```diff
@@ -0,0 +1,162 @@
+import { RuleTester } from '@typescript-eslint/rule-tester';
+import { rules } from '../../eslintPlugin';
+
+const ruleTester = new RuleTester({
+  parserOptions: {
+    ecmaFeatures: {
+      jsx: true,
+    },
+    sourceType: 'module',
+  },
+  parser: '@typescript-eslint/parser',
+});
+
+const animatedStyle = `
+const animatedStyle = useAnimatedStyle(() => {
+    return {
+      width: size,
+      height: size,
+    };
+  }, []);
+`;
+
+const animatedStyleError = {
+  messageId: 'animatedStyle',
+  data: {
+    componentName: 'View',
+    variableName: 'animatedStyle',
+  },
+} as const;
+
+const sharedValueError1 = {
+  messageId: 'sharedValue',
+  data: {
+    componentName: 'View',
+    propertyName: 'width',
+    propertyValue: 'widthSv',
+  },
+} as const;
+
+const sharedValueError2 = {
+  messageId: 'sharedValue',
+  data: {
+    componentName: 'View',
+    propertyName: 'width',
+    propertyValue: 'width',
+  },
+} as const;
+
+/**
+ * Correct code correctly classified as satisfying the rule
+ */
+const trueNegative = [
```

Comment:
I don't quite like the names of the tests now. Names of the test cases should mainly say what they are testing, not what is their outcome. It should something among the lines of

- non-animated component with non-animated style
- non-animated component with animated style
- non-animated component with shared value prop
- animated component with non-animated style
- etc.

Reply:
Unfortunately we don't have names of test cases when testing the eslint rules, the way we have in normal jest tests. Instead, we have the code snippet injected in its place.
I suppose you don't mean "names of the tests" but names of the variables storing the tests and additional description in comments. Then I understand that your suggestions have many benefits, but I would keep it the way it is for the following reasons:

* Variable names don't contain spaces, so `nonAnimatedComponentWithNonAnimatedStyle` maybe more descriptive, but `nonAnimatedComponentWithNonAnimatedStyle` and `nonAnimatedComponentWithAnimatedStyle` look similar and shorter names like `truePositive` and `trueNegative` are easier to work with
* It's intuitive that `trueNegative` are cases, that should be detected as breaking the rule, but they ere not
* The whole code snippet is acting like a test name, so it is easy to read that we are dealing, for example, with "non-animated component with animated style"

Reply:
I'm considering DX here.

Information that something broke in `falsePositive` doesn't really tell you much, you need to dive into it to get what exactly is failing.

On the other hand, if I want to look up something, for e.g. "what should I expect when I pass `animatedStyle` to a `ReaAnimated` component" I must look through all the test suites since I don't know if this case is even handled in the tests.

If you want to keep the `falseNegative` logic etc. at least put those tests in some domains. E.g.:

```ts
const withAnimatedComponent = {
  withAnimatedStyle: {
    truePositive: [...]
    falsePositive: [...]
    ...
  },
  withSharedValue: {...},
}

const withNonAnimatedComponent = {
  ...
};